estuary/api/v1.py

```diff
@@ -281,7 +282,21 @@ def get_siblings(resource, uid):
         serialized_node['resource_type'] = inflated_node.__label__
         serialized_results.append(serialized_node)
 
-    return jsonify(serialized_results)
+    if story_node.__label__ == DistGitCommit.__label__:
+        header = create_header(
+            story_node.__label__, story_node.hash_, story_node_story_flow, backward)
+    else:
+        header = create_header(
+            story_node.__label__, story_node.id_, story_node_story_flow, backward)
+
+    result = {
+        'data': serialized_results,
+        'meta': {
+            'siblings_table_header': header
```

Comment:
Could you rename this key to `description` or something along those lines? `siblings_table_header` is very specific and this could be useful for other things.